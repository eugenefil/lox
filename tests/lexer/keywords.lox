and
break
class
continue
else
false
fn
for
if
in
nil
or
return
super
this
true
var
while
