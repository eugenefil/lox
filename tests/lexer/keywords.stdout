And <none> "and"
Break <none> "break"
Class <none> "class"
Continue <none> "continue"
Else <none> "else"
False false "false"
Fn <none> "fn"
For <none> "for"
If <none> "if"
In <none> "in"
Nil <none> "nil"
Or <none> "or"
Return <none> "return"
Super <none> "super"
This <none> "this"
True true "true"
Var <none> "var"
While <none> "while"
